### DISCUSSION TITLE
Removing password requirements?

### DISCUSSION THREAD

**Discussion by Solidus-Prime** [kimai/kimai#2284] [Questions & Answers] [answered]
Hello,

Is there a way to completely remove the password requirement when creating a user? We would like to use the clock as a traditional kiosk where users just walk up and enter their ID to log in. 

If not, is there perhaps a way to make it so it requires less than 8 characters?

**Comment by kevinpapst** [comment, marked as answer]
Buy the Kiosk plugin:https://www.kimai.org/store/kiosk-barcode-bundle.html and support Kimai.

Users can define any arbitrary ID to start and stop tracking. Or you use a barcode/rfid scanner (see docs) .

**Reply by kevinpapst** [reply]
Otherwise: https://github.com/kevinpapst/kimai2/blob/master/config/validator/validation.yaml
But the normal Kimai UI is really not made for a Kiosk mode...

**Reply by Solidus-Prime** [reply]
Thanks, I put in a PO request for the Custom Fields and the Kiosk plugins with our front desk.
